src/app/[locale]/(app)/credits/_components/Flag.tsx

```diff
@@ -0,0 +1,17 @@
+import dynamic from "next/dynamic";
+
+export default function Flag({ code }: { code: string }) {
+  const Svg = dynamic(
+    () => import(`flagpack-core/svg/l/${code.toUpperCase()}.svg`),
```

Comment:
We'll have to be careful with the translation field in the contributors.json. For instance, `EN` does not exist but `GB-ENG`.
Ref: https://github.com/Yummygum/flagpack-core/tree/main/svg/l

Where to put this link?